app/api/api_v1/routers/pesquisa/pessoa.py

```diff
@@ -19,8 +19,7 @@
 async def pessoa_by_name(
     request: Request,
     pessoa_name: str,
-    db=Depends(get_db),
-    current_pessoa=Depends(get_current_active_pessoa),
```

Comment:
Se essa variável não for mais utilizada, remover o import da linha 12: `from app.core.auth import get_current_active_pessoa`